deploy-templates/values.yaml

```diff
@@ -80,6 +80,11 @@ gerrit:
     # --  For Kubernetes >= 1.18 you should specify the ingress-controller via the field ingressClassName
     # --  See https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#specifying-the-class-of-an-ingress
     # ingressClassName: nginx
+    # -- Defines the base URL for the gerrit.
+    # -- If not specified the pattern "gerrit-[namespace].[global DNS wildcard]" will create a URL like "gerrit-namespace.example.com"
+    host: ""
+    # -- Hostname(s) for the Ingress resource
+    # -- Ingress TLS configuration
```

Comment:
please, align these comments, for example https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L287